R/promptAndRun.R

```diff
@@ -77,27 +91,8 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
       message("Found ", length(myruns), if (mydir == "-a") " active", " runs.",
               if (length(myruns)/as.numeric(daysback) > 20) " Excuse me? You need a cluster only for yourself it seems.")
     }
-
-#    if (length(myruns)>40) {
-#        message("Excuse me? > 40 runs? You need a cluster only for yourself it seems")
-#    } else if (length(myruns)>15) {
-#        message("Please wait, I found more than 15 runs (wow)")
```

Comment:
this loop is exactly what `loopRuns` does, just without the color and with different sorting. To reduce redundancy, I added the color = FALSE and sortbytime = FALSE option to loopRuns and can therefore delete most of the stuff here.